run_dir/design/base.html

```diff
@@ -51,6 +51,7 @@
                         <li><a href="/reads_vs_qv">Reads vs QV</a></li>
                         <li><a href="/production/cronjobs">Cronjobs List</a></li>
                         <li><a href="/quotas">Quotas</a></li>
+                        <li><a href="/clusters_per_lane">Clusters per lane</a></li>
                         <li class="depreciated_page divider"></li>
                         <li id="depreciated_header"><a href="#">Depreciated Pages</a></li>
```

Comment:
@ewels @mariogiov  now that I see this and as we have been discussing it.... should this be depreciated or deprecated? :-/ actually, "untrustable" if that word exists, is what fits more to these pages.